penguin/cli/opencode_launcher.py

```
@@ -974,23 +1015,37 @@ def main(argv: Sequence[str] | None = None) -> int:
+            executable=Path(opencode_cmd[0]).name if opencode_cmd else "",
+            cwd=str(opencode_cwd) if opencode_cwd is not None else None,
+        )
     except RuntimeError as exc:
         if cleanup_registered:
             _stop_process(server_proc)
         print(f"Error: {exc}", file=sys.stderr)
         return 1
 
     try:
+        _profile_startup("launcher.spawn.start")
         result = subprocess.run(opencode_cmd, cwd=opencode_cwd, env=env)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)